content/versions-comparison/4.0-news.json

```diff
@@ -1,55 +1,48 @@
 [
   {
-    "title":"Assets structuration with families and attributes 🖼️",
+    "title":"Asset Manager for your digital assets 🖼️",
     "domains": ["Asset","Governance"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
   },
   {
-    "title":"Asset data enrichment 🖼️",
-    "domains": ["Asset","Quality"],
-    "editions": ["EE"],
-    "since-version": "4.0",
-    "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
-  },
-  {
-    "title":"Asset data contextualization by locale and channel",
+    "title":"Contextualize asset content by locale and channel",
     "domains": ["Asset","Quality"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#contextualize-your-asset-information"
   },
   {
-    "title":"External media used as product asset 🖼️",
+    "title":"Easily retrieve 3rd-party assets 🖼️",
     "domains": ["Asset","Governance"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
   },
   {
-    "title":"Asset completeness 🖼️",
+    "title":"Completeness on assets 🖼️",
     "domains": ["Asset","Quality"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#permissions-completeness"
   },
   {
-    "title":"Flexible and automatized asset transformations 🤖",
+    "title":"New asset transformations 🤖",
     "domains": ["Asset","Productivity"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#improved-asset-transformation-rules"
   },
   {
-    "title":"Efficient mass asset upload",
+    "title":"Improved mass asset upload",
     "domains": ["Asset","Productivity"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#mass-upload"
   },
   {
-    "title":"Automatic asset/product link 🤖",
+    "title":"Asset product link rules 🤖",
```

Comment:
I'm not talking about the rules of the rule engine here, but about the fact that we can link automatically asset to products based on some name format. This feature was heavily asked by our customers.